src/image-text/image-text.scss

```diff
@@ -46,6 +46,10 @@
     color: var(--text-color);
     width: 40%;
 
+    .text {
+      margin-top: 1rem;
```

Comment:
It might cause problems using rem when the screen is split into regions, and each region should scale properly.